### PR TITLE
Add ITE design power to Zone Summary report

### LINKED ISSUE #9671: ElectricEquipment:ITE:AirCooled design power not accounted for in the Zone Summary report
Issue overview
--------------
The design power for `ElectricEquipment:ITE:AirCooled` objects is not accounted for in the Zone Summary report.

### Details
Some additional details for this issue (if relevant):
 - Platform: Windows 10
 - Version of EnergyPlus: 22.1

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file: [defect_file.zip](https://github.com/NREL/EnergyPlus/files/9636268/defect_file.zip)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by lymereJ**
Pull request overview
---------------------
 - Fixes #9671

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [x] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [x] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/OutputReportTabular.cc [hunk 1]
```diff
@@ -11986,6 +11986,11 @@ void WriteVeriSumTable(EnergyPlusData &state)
                         totPlugProcess += state.dataHeatBal->ZoneHWEq(iPlugProc).DesignLevel;
                     }
                 }
+                for (iPlugProc = 1; iPlugProc <= state.dataHeatBal->TotITEquip; ++iPlugProc) {
+                    if (iZone == state.dataHeatBal->ZoneITEq(iPlugProc).ZonePtr) {
+                        totPlugProcess += state.dataHeatBal->ZoneITEq(iPlugProc).DesignTotalPower;
+                    }
+                }
```
Comment (lymereJ) [inline src/EnergyPlus/OutputReportTabular.cc:11993]: Add ITE total design power to sum of zone plug loads.

File: tst/EnergyPlus/unit/OutputReportTabular.unit.cc [hunk 2]
```diff
@@ -6460,20 +6473,33 @@ TEST_F(SQLiteFixture, WriteVeriSumTable_TestNotPartOfTotal)
...
+         (state->dataHeatBal->ZoneElectric(1).DesignLevel + state->dataHeatBal->ZoneITEq(1).DesignTotalPower) /
+             state->dataHeatBal->Zone(1).FloorArea},
+        {"Unconditioned Total",
+         "Plug and Process",
+         (state->dataHeatBal->ZoneElectric(2).DesignLevel + state->dataHeatBal->ZoneITEq(2).DesignTotalPower) /
+             state->dataHeatBal->Zone(2).FloorArea},
+        {"Not Part of Total",
+         "Plug and Process",
+         (state->dataHeatBal->ZoneElectric(3).DesignLevel + state->dataHeatBal->ZoneITEq(3).DesignTotalPower) /
+             state->dataHeatBal->Zone(3).FloorArea},
```
Comment (lymereJ) [inline tst/EnergyPlus/unit/OutputReportTabular.unit.cc:6502]: Update existing test to make sure that ITE power is included in the Zone Summary report.

File: src/EnergyPlus/OutputReportTabular.cc (hunk 1 above)
Comment (Myoldmopar) [reply src/EnergyPlus/OutputReportTabular.cc:11993]: Seems reasonable to me.

File: tst/EnergyPlus/unit/OutputReportTabular.unit.cc [hunk 3]
```diff
@@ -6315,6 +6318,16 @@ TEST_F(SQLiteFixture, WriteVeriSumTable_TestNotPartOfTotal)
...
 
+    state->dataHeatBal->ZoneITEq(1).ZonePtr = 1;
+    state->dataHeatBal->ZoneITEq(1).spaceIndex = 1;
+    state->dataHeatBal->ZoneITEq(1).DesignTotalPower = 750.0;
+    state->dataHeatBal->ZoneITEq(2).ZonePtr = 2;
+    state->dataHeatBal->ZoneITEq(2).spaceIndex = 2;
+    state->dataHeatBal->ZoneITEq(2).DesignTotalPower = 300.0;
+    state->dataHeatBal->ZoneITEq(3).ZonePtr = 3;
+    state->dataHeatBal->ZoneITEq(3).spaceIndex = 3;
+    state->dataHeatBal->ZoneITEq(3).DesignTotalPower = 150.0;
```
Comment (Myoldmopar) [inline tst/EnergyPlus/unit/OutputReportTabular.unit.cc:6329]: This makes sense.

File: tst/EnergyPlus/unit/OutputReportTabular.unit.cc (hunk 2 above)
Comment (Myoldmopar) [reply tst/EnergyPlus/unit/OutputReportTabular.unit.cc:6502]: This is fine, but it's pretty hard to read.  Next time, please do a quick cleanup replacing all the `state->dataHeatBal->ZoneElectric(1)` with `zElectric1` and similar to other long variable expressions.  Anyway, this is fine.

**Review by Myoldmopar — APPROVED** [review]
No issues from code review, testing locally.

File: src/EnergyPlus/OutputReportTabular.cc (hunk 1 above)
Comment (rraustad) [reply src/EnergyPlus/OutputReportTabular.cc:11993]: for (auto &zoneITEq : state.dataHeatBal->ZoneITEq)

Comment (Myoldmopar) [reply src/EnergyPlus/OutputReportTabular.cc:11993]: Definitely right on that @rraustad; I wasn't going to push on that for this quick last minute fix, but if this is delayed for any reason it should be cleaned up.

**Comment by Myoldmopar** [comment]
Local testing was successful.  Regressions are showing in just the ITE files, as expected.  And the results are an improvement.  In develop, the plug and process loads are zero and in this branch they are present.  I see no problem with this change and no reason to hold it up or wait for CI.  I'm going to merge, thanks @lymereJ

**Pull request merged**